src/Symfony/Component/Serializer/Tests/Mapping/Loader/AttributeLoaderTest.php

```diff
@@ -110,6 +110,8 @@ public function testLoadSerializedName()
         $attributesMetadata = $classMetadata->getAttributesMetadata();
         $this->assertEquals('baz', $attributesMetadata['foo']->getSerializedName());
         $this->assertEquals('qux', $attributesMetadata['bar']->getSerializedName());
+        $this->assertEquals('duxi', $attributesMetadata['duux']->getSerializedName());
+        $this->assertEquals('duxa', $attributesMetadata['duux']->getSerializedName(['a']));
```

Comment:
```suggestion
        $this->assertSame('duxi', $attributesMetadata['duux']->getSerializedName());
        $this->assertSame('duxa', $attributesMetadata['duux']->getSerializedName(['a']));
```

Also for the other possible ones in this test class

Reply:
I've changed only new assertions to not bloat already huge PR.